v0.6.15 coveralls-checksums.txt has incorrect path prefix (`release/`)

### What was wrong?

<!-- Incorrect report, unable to download the utility, an unexpected error -->

We noticed coverage runs via the github action failing because the path specified in the latest release's checksums file doesn't match the location of the binary in question. It looks like the latest release includes a `release/` prefix which isn't present in at the very least the last release.

### Logs

<!-- The output of coveralls utility with version and enabled --debug option if possible -->

Our CI run is at https://github.com/gittuf/gittuf/actions/runs/11147824919/job/30983463583?pr=578#step:5:33

```
  mkdir -p ~/bin/
  cd ~/bin/
  if [ $COVERAGE_REPORTER_VERSION == "latest" ]
  then
    asset_path=latest/download
  else
    asset_path="download/${COVERAGE_REPORTER_VERSION}"
  fi
  curl -sLO "[https://github.com/coverallsapp/coverage-reporter/releases/${asset_path}/coveralls-linux.tar.gz](https://github.com/coverallsapp/coverage-reporter/releases/$%7Basset_path%7D/coveralls-linux.tar.gz)"
  curl -sLO "[https://github.com/coverallsapp/coverage-reporter/releases/${asset_path}/coveralls-checksums.txt](https://github.com/coverallsapp/coverage-reporter/releases/$%7Basset_path%7D/coveralls-checksums.txt)"
  cat coveralls-checksums.txt | grep coveralls-linux.tar.gz | sha256sum --check
  tar -xzf coveralls-linux.tar.gz
  rm coveralls-checksums.txt
  echo ~/bin >> $GITHUB_PATH
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    COVERAGE_REPORTER_VERSION: latest
sha256sum: release/coveralls-linux.tar.gz: No such file or directory
sha256sum: WARNING: 1 listed file could not be read
release/coveralls-linux.tar.gz: FAILED open or read
Error: Process completed with exit code 1.
```

+1 this is also affecting our team today.

```
Run mkdir -p ~/bin/
  mkdir -p ~/bin/
  cd ~/bin/
  curl -sLO https://github.com/coverallsapp/coverage-reporter/releases/latest/download/coveralls-linux.tar.gz
  curl -sLO https://github.com/coverallsapp/coverage-reporter/releases/latest/download/coveralls-checksums.txt
  cat coveralls-checksums.txt | grep coveralls-linux.tar.gz | sha[2](https://github.com/dispel/dispel/actions/runs/11148756317/job/30986336387#step:6:2)56sum --check
  tar -xzf coveralls-linux.tar.gz
  rm coveralls-checksums.txt
  echo ~/bin >> $GITHUB_PATH
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
sha256sum: release/coveralls-linux.tar.gz: No such file or directory
release/coveralls-linux.tar.gz: FAILED open or read
sha256sum: WARNING: 1 listed file could not be read
Error: Process completed with exit code 1.
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -221,7 +221,7 @@ jobs:
           cp -r artifacts/coveralls-linux-binaries/* release/
           find artifacts/ -type f -exec cp \{} release/ \;
           mv release/coveralls.exe release/coveralls-windows.exe
-          sha256sum release/* > release/coveralls-checksums.txt
+          (cd release && sha256sum * > coveralls-checksums.txt)
 
       - name: List files in release directory (debug)
         run: |

